apps/gov-website/pages/api/identity/judgement.tsx

```diff
@@ -71,6 +71,9 @@ export default withMethodGuard(
 
 			return res.json({ ok: true });
 		} catch (error) {
+			if (error?.message === "DISCORD_USER_NOT_FOUND")
+				return res.json({ ok: true, message: error.message });
```

Comment:
OK approach but not ideal, this should come back as an error, and let the UI catch and handle that issue. will need to update this when doing #40